backend/fms_core/services/profile.py

```diff
@@ -0,0 +1,54 @@
+from collections import defaultdict
+from fms_core.models import Profile, FreezemanUser
+from fms_core.schema_validators import PREFERENCES_VALIDATOR
+
+def get_default_profile() -> Profile:
+    return Profile.objects.get(name="Default")
+
+def get_profile(user_id: int) -> Profile:
+    from fms_core.services.freezeman_user import get_or_create_freezeman_user
+    fm_user = get_or_create_freezeman_user(user_id)
```

Comment:
don't use get_or_create everywhere... you need to have cases where you would need to create the object if it does not exists (as a normal behaviour). Here if the Freezeman user does not exist we want it to crash and we want to investigate (it is not normal that the user does not exists).